crates/floresta-wire/src/p2p_wire/error.rs

```diff
@@ -114,3 +117,19 @@ impl Display for AddrParseError {
         }
     }
 }
+
+#[derive(Debug, Clone)]
+pub enum AddnodeError {
+    AlreadyExists(IpAddr, u16),
```

Comment:
/// Tried to add an address that is either already connected, or added by the user